src/actions/backlogItems.ts

```diff
@@ -1,13 +1,18 @@
 // middleware
 import { API, ApiAction } from "../middleware/apiMiddleware";
+
+// actions
 import * as ActionTypes from "./actionTypes";
 
+// consts/enums
+import { APPLICATION_JSON } from "../constants";
+
 export const getBacklogItems = (): ApiAction<undefined> => ({
     type: API,
     payload: {
         endpoint: "http://localhost:8500/api/v1/backlog-items",
         method: "GET",
-        headers: { "Content-Type": "application/json", Accept: "applicatio/json" },
+        headers: { "Content-Type": APPLICATION_JSON, Accept: APPLICATION_JSON },
```

Comment:
using constants will make sure that this type of spelling mistake doesn't get into the code again